app/components/Dashboard.tsx

```diff
@@ -0,0 +1,159 @@
+        router.push(
+          getProjectRevisionPageRoute(response.createProject.projectRevision.id)
+        );
+      },
+    });
+  }, [createProject, isProjectCreating, router]);
+
+  const isAdmin = useMemo(
+    () => ADMIN_ROLES.some((role) => session?.userGroups?.includes(role)),
+    [session?.userGroups]
+  );
+
+  const addOrResumeProjectLink = useMemo(
+    () =>
+      pendingNewProjectRevision ? (
+        <Link
+          passHref
+          href={getProjectRevisionPageRoute(pendingNewProjectRevision.id)}
+        >
+          <BCGovLink>Resume Project Draft</BCGovLink>
+        </Link>
+      ) : (
+        <button onClick={handleProjectCreation} disabled={isProjectCreating}>
+          Create a new Project
+        </button>
+      ),
+    [pendingNewProjectRevision, handleProjectCreation, isProjectCreating]
+  );
+
+  return (
+    <>
+      <header>
+        <h2>Welcome, {session.cifUserBySub?.firstName}</h2>
+      </header>
+      <div>
+        <section>
+          <h3 id="projects-heading">Projects</h3>
+          <nav aria-labelledby="projects-heading">
+            <Link passHref href={getProjectsPageRoute()}>
+              <BCGovLink>Projects List</BCGovLink>
+            </Link>
+            {addOrResumeProjectLink}
+          </nav>
+        </section>
+        <section>
+          <h3 id="reporting-op-heading">Reporting Operations</h3>
+          <nav aria-labelledby="reporting-op-heading">
+            <Link passHref href={getOperatorsPageRoute()}>
+              <BCGovLink>Operators</BCGovLink>
+            </Link>
+            <Link passHref href={getContactsPageRoute()}>
+              <BCGovLink>Contacts</BCGovLink>
+            </Link>
+          </nav>
+        </section>
+        {isAdmin && (
+          <section>
+            <h3 id="admin-heading">Administration</h3>
+            <nav aria-labelledby="admin-heading"></nav>
+          </section>
+        )}
+      </div>
+      <style jsx>{`
+        div,
+        nav {
+          display: flex;
+          flex-wrap: wrap;
+        }
+
+        section {
+          width: 18rem;
+          margin: 1rem;
+          border: 1px solid #939393;
+          border-radius: 4px;
+        }
+
+        section > h3 {
+          min-height: 5rem;
+          margin-bottom: 0;
+        }
+
+        :global(section > nav > *),
+        section > h3 {
+          padding: 0.75rem 1.25rem;
+        }
+
+        section > nav {
+          flex-direction: column;
+        }
+
+        :global(section > nav > *:not(:last-child)),
+        section > h3 {
+          border-bottom: 1px solid #939393;
+        }
+
+        :global(button) {
+          color: #1a5a96;
+          background: transparent;
+          border: none;
+          cursor: pointer;
+          text-decoration: underline;
+          text-align: left;
+        }
+        :global(button:hover) {
+          text-decoration: none;
+          color: blue;
```

Comment:
the BC design system uses #0056b3 for the hover color...but I couldn't find this actually written anywhere

Reply:
This button is displayed as a link, so it's blue for the hover: https://developer.gov.bc.ca/Design-System/Links